lib/tapioca/cli.rb

```diff
@@ -21,10 +21,53 @@ class Cli < Thor
       desc: "Verbose output for debugging purposes",
       default: false
 
-    desc "init", "initializes folder structure"
-    option :postrequire, type: :string, default: DEFAULT_POSTREQUIRE_FILE
+    desc "init", "get project ready for type checking"
     def init
-      command = Commands::Init.new(
+      configure_command = Commands::Configure.new(
```

Comment:
Why isn't this command just calling the other methods instead of reimplementing them?

```ruby
invoke :configure
invoke :annotations
invoke :gem
invoke :todo
```